wallet/how-to/use-non-evm-networks/starknet/index.md

```diff
@@ -0,0 +1,8 @@
+---
+description: Interact with users' Starknet accounts in MetaMask.
+sidebar_position: 1
+---
+
+# Starknet
+
+You can interact with users' Starknet accounts in MetaMask by installing the Starknet Snap.
```

Comment:
I think a better way to say this is, "by connecting to the Starknet Snap for MetaMask."